### DISCUSSION TITLE
[Assets] Assets select not working

### DISCUSSION THREAD

**Discussion by RaresVlaiduc** [GrapesJS/grapesjs#4078] [Q&A]
Hello everybody!

I am trying to execute a function when the "asset:select" event is triggered, but it does't work.
I open the assets manager modal, add an image and when I select the image, nothing happens.. the function is not called.

Can someone tell me exactly what am I doing wrong?

v0.17.27

**Comment by artf** [comment]
This is the list of available events https://grapesjs.com/docs/api/assets.html#available-events
If you don't see it here, it doesn't exist (don't rely on `Assets.events`, it's not a public/documented property)
